post-workshop/post-workshop-ja.md

```diff
@@ -1,52 +1,52 @@
+*機密保持*
+回答は匿名で記録され、固有のIDを作成するように求められます。ワークショップの事前アンケートに回答した場合は、同じIDを使用してください。 なお、これらの匿名および集計された回答は、学習者の経験を理解し、今後のワークショップを改善し、インストラクターが教育計画に反映するための詳細なフィードバックを提供する目的で、ワークショップのホスト、インストラクター、ヘルパー、カーペントリーズ・コア・チームと共有されます。
 
 
-*Risks and Benefits*
-There are no direct risks or benefits to you from filling out this survey, and no compensation. We hope to use these results to improve workshops for future learners.
+*リスクとメリット*
+このアンケートにご協力いただくことによる直接的なリスクやメリットはなく、報酬もありません。この結果をもとに、今後の学習者のため、ワークショップの改善に役立てたいと考えています。
 
-*Consent*
-You are not required to take this survey to participate in our workshop. You may quit the survey at any time or skip any item other than those required to correctly sort your responses. 
+*承諾*
+このアンケートに回答することは、ワークショップに参加する条件ではありません。アンケートはいつ終了しても構いませんし、回答できる項目以外はスキップしていただいても構いません。
 
-If you have any questions about the study, please contact Erin Becker, Associate Director of The Carpentries at ebecker@carpentries.org or eribecker@ucdavis.edu or Megan Welsh, Assistant Professor of Education at the University of California, Davis at megwelsh@ucdavis.edu.
+この研究についてご質問がある場合は、Carpentriesのアソシエイト・ディレクターであるエリン・ベッカー（ebecker@carpentries.org または eribecker@ucdavis.edu）か、またはカリフォルニア大学デービス校の教育学助教授であるミーガン・ウェルシュ（megwelsh@ucdavis.edu）までご連絡ください。
 
-If you have any questions about your rights or treatment as a research participant in this study or would like to provide input about this research, please contact the University of California at Davis’ Institutional Review Board (IRB) at (916) 703-9151, IRBAdmin@ucdmc.ucdavis.edu, or 2921 Stockton Blvd, Suite 1400, Room 1429, Sacramento, CA 95817.
+本研究の参加者としての権利や待遇について質問がある場合、または本研究について意見を提供したい場合は、カリフォルニア大学デービス校のInstitutional Review Board（IRB）（電話：(916) 703-9151、IRBAdmin@ucdmc.ucdavis.edu、または2921 Stockton Blvd, Suite 1400, Room 1429, Sacramento, CA 95817）までご連絡ください。
 
 -----------
 
-### Do you consent to taking this survey? *
+### このアンケートを受けることに同意しますか？ *
 
-   - yes
-   - no
+   - はい
+   - いいえ
 
-### Are you 18 years of age or above? *
+### 18歳以上の方ですか？ *
 
-   - yes
-   - no
+   - はい
+   - いいえ
 
-### What is the identifier for your workshop? It is in the YYYY-MM-DD-sitename format and is used in the URL for the workshop website or the Etherpad you used.  Please copy and paste the workshop id into the text box below.  *
+### ワークショップのIDは何ですか？YYYY-MM-DD-sitename形式で、ワークショップのウェブサイトまたは使用したEtherpadのURLに使用されています。 以下のテキストボックスにワークショップIDをコピーして貼り付けてください。 *
 
-(short text answer)
+(短文回答)
 
-### Please enter a unique identifier as follows: Number of siblings (as numeric) + First two letters of the city you were born in (lowercase) + First three letters of your current street (lowercase).  
+### 固有のIDを次のように入力してください：兄弟・姉妹の人数（半角数字）+ 生まれた街の最初の2文字（ローマ字の小文字）+ 現在住んでいる通りの最初の3文字 (ローマ字の小文字)
```

Comment:
```suggestion
### 固有の ID を以下のように入力してください。兄弟・姉妹の人数（半角数字）+ 生まれた街の最初の2文字（小文字）＋現在住んでいる通りの最初の3文字 (小文字)。
```
Update for consistency with @masamiy's translation in #6. We will review the rest of the survey and merge with a literal translation of the English version for now. After discussion with The Carpentries organisation, we will revisit the issue for how to adapt the anonymous unique memorable identifier system to participants living in Japan. Thanks for raising this issue @naganeayu!